mentors/Dockerfile

```diff
@@ -12,5 +12,6 @@ RUN pip3 install -r /tmp/requirements.txt && \
     rm /tmp/requirements.txt
 
 COPY src /app/src
+COPY secrets /app/secrets
```

Comment:
Probably we should never copy the secrets into the docker image and instead make the scripts look for them in env variables. The local `mentor/secrets` files can then be read in by the makefile and passed when this is run locally. If it's ever run in CI, we can set the env vars in the CI project.

Reply:
Oh, duh